R/pull_acs.R

```diff
@@ -43,5 +45,18 @@ df %>%
 # use clean_acs to get data with descriptive labels -------
 df <- clean_acs(state = st, county = co, 
               geography = geo, variables = var, 
-              year = yr,
+              year = yr, span = span,
               key = key)
+
+# ... set variables to NULL and use table to get a whole table
+df <- clean_acs(state = st, county = co, 
```

Comment:
if you set "variables" to NULL and add a"table = [one table code]" argument, you can pull an entire table with a single code instead of having to enumerate variable codes for each value in the table